docs/init-config.sample.yaml

```diff
@@ -41,72 +41,71 @@ LCoreAlloc:
 
 # FACE CREATION
 Face:
-  # whether to enable Ethernet faces
+  # Whether to enable Ethernet faces.
   EnableEth: true
-  # whether to disable RxFlow dispatching
+  # Whether to disable RxFlow dispatching.
   EthDisableRxFlow: false
-  # Ethernet device MTU
+  # Ethernet device MTU.
   EthMtu: 1500
-  # Ethernet RX queue capacity
+  # RX queue capacity for Ethernet faces.
   EthRxqFrames: 4096
-  # Ethernet before-TX queue capacity
+  # Before-TX queue capacity for Ethernet faces.
   EthTxqPkts: 256
-  # Ethernet after-TX queue capacity
+  # After-TX queue capacity for Ethernet faces.
   EthTxqFrames: 4096
 
-  # whether to enable socket faces
+  # Whether to enable socket faces.
   EnableSock: true
-  # socket before-TX queue capacity
+  # Before-TX queue capacity for socket faces.
   SockTxqPkts: 256
-  # socket after-TX queue capacity
+  # After-TX queue capacity for socket faces.
   SockTxqFrames: 1024
 
-  # ChanRxGroup queue capacity (shared among all socket/mock faces)
+  # ChanRxGroup queue capacity, shared among all socket/mock faces.
   ChanRxgFrames: 4096
 
-# NDT
+# NAME DISPATCH TABLE
 Ndt:
-  # Names are dispatched using prefix with this number of components.
+  # Names are dispatched using a prefix with this number of components.
   PrefixLen: 2
-  # There are (2^IndexBits) elements.
+  # There are 2^IndexBits elements in the table.
   IndexBits: 16
-  # Collect per-element sample every (2^SampleFreq) lookups. Must not exceed 30.
+  # Collect per-element sample every 2^SampleFreq lookups. Must not exceed 30.
```

Comment:
I'm not quite sure what this means, can you explain so that I can provide a better description?

Reply:
Each NDT entry has a 16-bit counter that indicates how often an Interest was dispatched using this NDT entry.
Since the 16-bit counter is quite narrow, the NDT lookup procedure only increments the per-entry counter, if a per-thread counter is multiple of 2^SampleFreq.

Reply:
So this is really the inverse of frequency, that "Freq" in the name is misleading.